### PR TITLE
fix StaticFiltersProvider styling

### PR DESCRIPTION
**Pull request by yen-tt**
update className's default styling from `md-56` to `md:w-56`

J=SLAP-2207
TEST=manual

include only StaticFilters on the left hand side of people page. With `md:w-56`, see that the width of it now changes when the page size changes from >768px to <768px

### CODE REVIEW

File: src/components/Filters/StaticFiltersProvider.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export type StaticFiltersProviderProps = PropsWithChildren<{
  */
 export function StaticFiltersProvider({
   children,
-  className = 'md-56',
```
Comment (oshi97) [inline src/components/Filters/StaticFiltersProvider.tsx:31]: for my edification what was md-56 doing before? did we just miss a breaking change when we upgraded to tailwind 3 or something?

Comment (oshi97) [reply src/components/Filters/StaticFiltersProvider.tsx:31]: tried looking for it but couldn't find anything :(

Comment (yen-tt) [reply src/components/Filters/StaticFiltersProvider.tsx:31]: I believe this was a typo. `md-56` does not apply any additional css to the div. md sets breakpoint (768px) of when the styling on the right hand side of the colon should be applied.
Screen Shot 2022-07-06 at 11 45 35 AM

Comment (oshi97) [reply src/components/Filters/StaticFiltersProvider.tsx:31]: awesome!

**Review by oshi97 — APPROVED** [review]

**Pull request merged**